pkg/logs/decoder/decoder.go

```diff
@@ -108,23 +105,11 @@ func InitializeDecoder(source *config.LogSource, parser parsers.Parser) *Decoder
 // NewDecoderWithEndLineMatcher initialize a decoder with given endline strategy.
 func NewDecoderWithEndLineMatcher(source *config.LogSource, parser parsers.Parser, matcher EndLineMatcher, multiLinePattern *regexp.Regexp) *Decoder {
 	inputChan := make(chan *Input)
-	brokenLineChan := make(chan *DecodedInput)
 	lineParserOut := make(chan *Message)
-	outputChan := make(chan *Message)
+	outputChan := make(chan *Message, 10)
```

Comment:
Just curious - why make this channel buffered now and why pick `10`? 
It may be worth defining this in `pkg/logs/config/constants.go` with the other global channel size configs.

Reply:
Huh, good question!  I have no recollection of why I did that.  I've reverted it now.